PKHeX.WinForms/Subforms/PKM Editors/MemoryAmie.cs

```diff
@@ -435,10 +435,10 @@ private void changeCountryText(object sender, EventArgs e)
 
         private void update255_MTB(object sender, EventArgs e)
         {
-            MaskedTextBox mtb = sender as MaskedTextBox;
+            MaskedTextBox mtb = (MaskedTextBox) sender;
```

Comment:
That's an article written in 2004, in .NET Frameworks 2.0+, type conversion gained a significant performance increase. If the performance decrease is negligible and we can avoid using conditional access to prevent null dereferences, then I think in the long run that is the much better solution.

Since I don't see too many current benches, might as well do my own and post results, because I still see debate on this too.